docs/source/conf.py

```diff
@@ -50,26 +49,6 @@
 project = 'traitsui'
 copyright = '2008-2020, Enthought'
 
-# The default replacements for |version| and |release|, also used in various
-# other places throughout the built documents.
-#
-# Pull from the actual release number without imports
-d = {}
-try:
-    version_path = os.path.join(base_path, '..', '..', 'traitsui', '_version.py')
-    with open(version_path, 'r', encoding='utf8') as fp:
-        exec(compile(fp.read(), version_path, 'exec'), d)
-    release = d['version']
-    version = '.'.join(d['version'].split('.', 2)[:2])
```

Comment:
Other ets packages still seem to be following this approach.

Reply:
Yeah. I need to fix that. I am not sure why we were following this approach - i'll ask first on the internal slack channel and then gut it from ETS projects.